backend/services/implementations/testSessionService.ts

```diff
@@ -187,6 +204,78 @@ class TestSessionService implements ITestSessionService {
 
     return testSessionDtos;
   }
+
+  /*
+   * computeTestGrades computes the breakdown and score of a given result and returns the ResultDTO
+   */
+  async computeTestGrades(
+    result: NewResultDTO,
+    testId: string,
+  ): Promise<ResultDTO> {
+    let resultDto: ResultDTO;
```

Comment:
Re: similar to the comment made in `interfaces/testSessionService.ts` let's call this `markedResult` or `gradedResult` or something along those lines.